sos/report/plugins/sunbeam_hypervisor.py

```diff
@@ -0,0 +1,114 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+
+class SunbeamHypervisor(Plugin, UbuntuPlugin):
+
+    short_desc = "Sunbeam Hypervisor snap"
+
+    plugin_name = "sunbeam_hypervisor"
+    profiles = ('cloud',)
+    packages = ('openstack-hypervisor',)
+
+    common_dir = '/var/snap/openstack-hypervisor/common'
+
+    def setup(self):
+
+        self.add_service_status([
```

Comment:
Can we have 'snap.openstack-hypervisor.*' instead? Things might change name/get added/removed, would be nice to capture all of it if possible.

Reply:
yup, makes sense, I'll amend that